exasol/ds/sandbox/runtime/ansible/reset_password_tasks.yml

```diff
@@ -1,16 +1,16 @@
-- name: set new default password
+- name: Set New Default Password
   ansible.builtin.user:
     name: "{{ansible_user}}"
     state: present
-    password: "{{ default_vm_password | password_hash('sha512') }}"
+    password: "{{ default_vm_password }}"
```

Comment:
Maybe, we should rename this variable to default_vm_password_hash

Reply:
Yes, see next push.